### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, please review this PR.

Required: Build an interactive console app.

Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

UI

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,49 +1,84 @@
...
+
+def create_person(app)
+  print 'Enter person type (student/teacher): '
+  type = gets.chomp.downcase
+  print 'Enter name: '
+  name = gets.chomp
+  print 'Enter age: '
+  age = gets.chomp.to_i
+  app.create_person(type, name, age)
+end
```
Comment (codepantha) [inline main.rb]: - [ ] Please ensure that your app UI closely mirrors the one in the example video. While creating a person, let the user choose between creating a teacher or a student by providing an input of 1 or 2 as shown below 👇 
### UI guideline
image
### Your UI
image

File: main.rb [hunk 2]
```diff
@@ -1,49 +1,84 @@
...
+
+def create_rental(app)
+  print 'Enter person ID: '
+  person_id = gets.chomp.to_i
+  print 'Enter book ID: '
+  book_id = gets.chomp.to_i
+  print 'Enter rental date: '
+  date = gets.chomp
+  app.create_rental(person_id, book_id, date)
+end
```
Comment (codepantha) [inline main.rb]: - [x] Please let your UI for creating a rental behave like the one in the guideline [here](https://www.youtube.com/watch?v=vkkgrhD6aXQ). Select the book and person from the list by number not by ID 👍 
### Guideline
image
### Your app
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @egichuhis 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 2 above)
Comment (egichuhis) [reply main.rb]: Thanks @codepantha for noting the above

**Review by ademibowale — APPROVED** [review]
Hi @egichuhis  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- created app.rb file that will serve as their console app entry-point✔️
- created `main.rb` the entry point✔️
- Descriptive commit messages ✔️ 
- the student's school library console app behaves in the same was as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) it was provided for them.✔️
- Good use of Gitflow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**